ydb/core/viewer/viewer.cpp

```diff
@@ -567,6 +585,56 @@ class TViewer : public TActorBootstrapped<TViewer>, public IViewer {
         }
         Send(ev->Sender, new NMon::TEvHttpInfoRes(GetHTTPNOTFOUND(ev->Get()), 0, NMon::IEvHttpInfoRes::EContentType::Custom));
     }
+
+    void Handle(NHttp::TEvHttpProxy::TEvHttpIncomingRequest::TPtr& ev) {
+        if (ev->Get()->Request->Method == "OPTIONS") {
+            TString uri(ev->Get()->Request->GetURI());
+            TString type = mimetypeByExt(uri.c_str());
+            if (type.empty()) {
+                type = "application/json";
+            }
+            TStringBuilder response;
+            response << "HTTP/1.1 204 No Content\r\n";
+            FillCORS(response, ev->Get());
+            response << "Content-Type: " + type + "\r\n"
```

Comment:
you really want to create temporary TString from "Content-Type: " + type + "\r\n" and then output it into stream and dispose after?